generated/nirfmxdemod/nirfmxdemod.proto

```diff
@@ -13,22 +13,2029 @@ option csharp_namespace = "NationalInstruments.Grpc.NiRFmxDemod";
+  NIRFMXDEMOD_ATTRIBUTE_IQ_POWER_EDGE_TRIGGER_SOURCE = 2105351;
+  NIRFMXDEMOD_ATTRIBUTE_IQ_POWER_EDGE_TRIGGER_LEVEL = 2105352;
+  NIRFMXDEMOD_ATTRIBUTE_IQ_POWER_EDGE_TRIGGER_SLOPE = 2105353;
+  NIRFMXDEMOD_ATTRIBUTE_TRIGGER_DELAY = 2105354;
+  NIRFMXDEMOD_ATTRIBUTE_TRIGGER_MINIMUM_QUIET_TIME_MODE = 2105355;
+  NIRFMXDEMOD_ATTRIBUTE_TRIGGER_MINIMUM_QUIET_TIME_DURATION = 2105356;
+  NIRFMXDEMOD_ATTRIBUTE_AUTO_LEVEL_INITIAL_REFERENCE_LEVEL = 2105357;
+  NIRFMXDEMOD_ATTRIBUTE_LIMITED_CONFIGURATION_CHANGE = 2105358;
+  NIRFMXDEMOD_ATTRIBUTE_REFERENCE_LEVEL_HEADROOM = 2109436;
+  NIRFMXDEMOD_ATTRIBUTE_SELECTED_PORTS = 2109437;
+  NIRFMXDEMOD_ATTRIBUTE_IQ_POWER_EDGE_TRIGGER_LEVEL_TYPE = 2109439;
+  NIRFMXDEMOD_ATTRIBUTE_RESULT_FETCH_TIMEOUT = 2109440;
+}
+
+enum ADemodAMCarrierSuppressedEnabled {
+  A_DEMOD_AM_CARRIER_SUPPRESSED_ENABLED_FALSE = 0;
+  A_DEMOD_AM_CARRIER_SUPPRESSED_ENABLED_TRUE = 1;
+}
+
+enum ADemodAudioFilterType {
+  A_DEMOD_AUDIO_FILTER_TYPE_NONE = 0;
+  A_DEMOD_AUDIO_FILTER_TYPE_CUSTOM = 1;
+  A_DEMOD_AUDIO_FILTER_TYPE_A_WEIGHT = 2;
+  A_DEMOD_AUDIO_FILTER_TYPE_B_WEIGHT = 3;
+  A_DEMOD_AUDIO_FILTER_TYPE_C_WEIGHT = 4;
+  A_DEMOD_AUDIO_FILTER_TYPE_CCITT = 5;
+  A_DEMOD_AUDIO_FILTER_TYPE_ITU_R_468_4 = 6;
+}
+
+enum ADemodAveragingEnabled {
+  A_DEMOD_AVERAGING_ENABLED_FALSE = 0;
+  A_DEMOD_AVERAGING_ENABLED_TRUE = 1;
+}
+
+enum ADemodAveragingType {
+  A_DEMOD_AVERAGING_TYPE_LINEAR = 0;
+  A_DEMOD_AVERAGING_TYPE_MAXIMUM = 1;
+  A_DEMOD_AVERAGING_TYPE_MINIMUM = 2;
+}
+
+enum ADemodCarrierFrequencyCorrectionEnabled {
+  A_DEMOD_CARRIER_FREQUENCY_CORRECTION_ENABLED_FALSE = 0;
+  A_DEMOD_CARRIER_FREQUENCY_CORRECTION_ENABLED_TRUE = 1;
+}
+
+enum ADemodCarrierPhaseCorrectionEnabled {
+  A_DEMOD_CARRIER_PHASE_CORRECTION_ENABLED_FALSE = 0;
+  A_DEMOD_CARRIER_PHASE_CORRECTION_ENABLED_TRUE = 1;
+}
+
+enum ADemodModulationType {
+  A_DEMOD_MODULATION_TYPE_AM = 0;
+  A_DEMOD_MODULATION_TYPE_FM = 1;
+  A_DEMOD_MODULATION_TYPE_PM = 2;
+}
+
+enum ADemodRbwFilterType {
+  A_DEMOD_RBW_FILTER_TYPE_NONE = 0;
+  A_DEMOD_RBW_FILTER_TYPE_GAUSSIAN = 1;
+  A_DEMOD_RBW_FILTER_TYPE_FLAT = 2;
+  A_DEMOD_RBW_FILTER_TYPE_SYNCH_TUNED_4 = 3;
+  A_DEMOD_RBW_FILTER_TYPE_SYNCH_TUNED_5 = 4;
+  A_DEMOD_RBW_FILTER_TYPE_RRC = 5;
+}
+
+enum DDemodAveragingEnabled {
+  D_DEMOD_AVERAGING_ENABLED_FALSE = 0;
+  D_DEMOD_AVERAGING_ENABLED_TRUE = 1;
+}
+
+enum DDemodDifferentialEnabled {
+  D_DEMOD_DIFFERENTIAL_ENABLED_FALSE = 0;
+  D_DEMOD_DIFFERENTIAL_ENABLED_TRUE = 1;
+}
+
+enum DDemodEqualizerMode {
+  D_DEMOD_EQUALIZER_MODE_OFF = 0;
+  D_DEMOD_EQUALIZER_MODE_TRAIN = 1;
+  D_DEMOD_EQUALIZER_MODE_HOLD = 2;
+}
+
+enum DDemodEvmNormalizationReference {
+  D_DEMOD_EVM_NORMALIZATION_REFERENCE_PEAK = 0;
+  D_DEMOD_EVM_NORMALIZATION_REFERENCE_RMS = 1;
+}
+
+enum DDemodFskReferenceCompensationEnabled {
+  D_DEMOD_FSK_REFERENCE_COMPENSATION_ENABLED_FALSE = 0;
+  D_DEMOD_FSK_REFERENCE_COMPENSATION_ENABLED_TRUE = 1;
+}
+
+enum DDemodM {
```

Comment:
Should circle back when checking / beautifying the API and see if this should be used as a bitfield_enum like we did for `MeasurementTypes` in RFmx DMA2K . They might be intended as distinct options (never combined) and then how it is here would be fine.

Reply:
Thanks for your comment. I will note this point in the Review/beautifying task.